app/models/dropdown_option.rb

```diff
@@ -10,6 +10,6 @@
 #
 
 class DropdownOption < ApplicationRecord
-  has_one :child_question, class_name: 'Question', as: :parent_option, :dependent => :destroy
+  has_many :child_questions, class_name: 'Question', as: :parent_option, :dependent => :destroy
```

Comment:
Why did this change? Does this change apply to `RangeOption` too?

Reply:
wait i have no clue why this has changed wtf

Reply:
I can like. undo this for now?